docs/config_search.md

```diff
@@ -255,9 +255,9 @@ profile_models:
 
 ---
 
-### **Limiting Batch Size, Instance Group, and Client Concurrency**
+### **Limiting Batch Size, Instance Group, and Client Concurrency or Request Rate**
```

Comment:
Isn't there a line further up the file to list run-config-search-min/max-request-rate?

Reply:
Yes, but this is in relation to quick search. Which I realized I shouldn't have changed, since we don't support request rate for quick search. I've removed this edit.